src/containers/App.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable no-param-reassign */
+import React, { PureComponent } from 'react';
+import styled from 'styled-components';
+import { moviesData, filtersData } from '../data';
+import Header from '../components/Header';
+import MoviesResult from './MoviesResult';
+import Footer from '../components/Footer';
+import ErrorBoundary from './ErrorBounding';
+
+class App extends PureComponent {
+    // eslint-disable-next-line react/state-in-constructor
+    state = {
+        searchValue: '',
+        sortValue: 'date',
+        filters: filtersData,
+        all: true,
+        movies: moviesData
+    }

```

Comment:
For mocking date better to use one of this case:
1. Pass data into component
```jsx
<App filters={filtersData} movies={moviesData} />
```
2. Using life cycle and mock request
```jsx
const data = { filters: filtersData, movies: moviesData };
...
componentDidMount() {
    Promise.resolve(() => ()).then((data) => this.setState({ ...this.state, ...data }))
}
```